cross/java-11-openjdk/Makefile

```
@@ -10,6 +10,9 @@ PKG_DIR = jdk11u-jdk-$(PKG_VERS).$(JAVA_MINOR)-$(JAVA_RELEASE)
 
 DEPENDS = cross/cups cross/fontconfig cross/expat
 BUILD_DEPENDS = cross/libX11 cross/libXrender cross/libXtst cross/libXt cross/libXrandr cross/alsa-lib cross/zip
+ifeq ($(findstring $(ARCH),hi3535 qotiq 88f6281),$(ARCH))
+BUILD_DEPENDS += cross/libffi
+endif
```

you can use predefined archs like this (and avoid typos like qotic -> qoriq)
hi3535 -> $(ARMv7L_ARCHS)
qoriq -> $(PPC_ARCHS)
88f6281 -> $(ARMv5_ARCHS)

```suggestion
include ../../mk/spksrc.archs.mk
ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(ARMv7L_ARCHS) $(PPC_ARCHS)),$(ARCH))
BUILD_DEPENDS += cross/libffi
endif
```

---

ok will do.